dashboard/src/actions/repos.test.tsx

```diff
@@ -735,7 +737,7 @@ describe("validateRepo", () => {
 
   it("dispatches checkRepo and errorRepos when the validation cannot be parsed", async () => {
     AppRepository.validate = jest.fn(() => {
-      return { statusCode: 409 };
+      return { error: "forbidden" };
```

Comment:
Shouldn't this be returning a `{ code: 409, message: "forbidden" }`? EDIT: Ah, the point is that the error is the same one in that is stringified in the payload below, was just confused by the unexpected data.

Reply:
right, I keep setting dummy data in the tests but it's true that it's more readable with a real result.